elm/Main.elm

```diff
@@ -197,16 +215,15 @@ update msg model =
             )
 
         Seek id s ->
-            let
-                c = if model.dragging == Just id then
-                        wsSend model.wsURL <| cmdSeek s
-                    else
-                        Cmd.none
-            in
-                ( { model | dragging = Nothing }, c )
+            ( { model | seek = Wait s }
+            , if model.seek == Drag id then
```

Comment:
If this is false seek will always stay `Wait s`?

Reply:
I don't know. It might not make sense the way it is. But the general idea should be:

* While dragging, let the control handle itself (except maybe if general bugginess requires us to always provide a value, in which case store the current drag position in the model already and draw from that. (Dragging state)
* After dragging, send out the command and make the control stay put at the sent value. (Wait state)
* After status update (while not dragging), go back to showing the server+delta value (Display state)